src/components/HealthCardDisplay/HealthCardDisplay.js

```diff
@@ -339,11 +379,47 @@ const HealthCardDisplay = () => {
                       {patientData.immunizations.map((item) => (
                         <div key={item.fullUrl}>
                           <ListItem>
-                            <HealthCardVaccination immunization={item.resource} />
+                            <HealthCardVaccination
+                              immunization={item.resource}
+                            />
                           </ListItem>
                         </div>
                       ))}
                     </List>
+                    <Grid
+                      container
+                      xs={10.5}
+                      justifyContent="left"
+                      alignItems="flex-end"
+                    >
+                      <Box
+                        display="flex"
+                        alignItems="center"
+                        justifyContent="left"
+                        flexDirection="row"
+                      >
+                        <Typography>{issuerDisplayName}</Typography>
```

Comment:
Would make this font bold, it looks a little anemic. Our other non-label data is also bold so we should keep it consistent.